### PR TITLE
Add new keyboard otd_plus

### PR DESCRIPTION
**Pull request by gadzkun**
## Description

add new keyboard from hardlineworks, otd_plus

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hardlineworks/otd_plus/config.h [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED  // "HLW"
```
Comment (zvecr) [inline keyboards/hardlineworks/otd_plus/config.h:6]: You want to do something to match the comment here, and avoid `0xFEED`
```suggestion
#define VENDOR_ID       0x484C  // "HL"
```

File: keyboards/hardlineworks/otd_plus/rules.mk [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/hardlineworks/otd_plus/rules.mk:11]:
```suggestion
```

File: keyboards/hardlineworks/otd_plus/otd_plus.h [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_all( \
```
Comment (fauxpark) [inline keyboards/hardlineworks/otd_plus/otd_plus.h:14]:
```suggestion
#define LAYOUT_tkl_ansi_wkl( \
```

File: keyboards/hardlineworks/otd_plus/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [0] = LAYOUT_all(
```
Comment (fauxpark) [inline keyboards/hardlineworks/otd_plus/keymaps/default/keymap.c:5]:
```suggestion
  [0] = LAYOUT_tkl_ansi_wkl(
```

File: keyboards/hardlineworks/otd_plus/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    KC_ESC,             KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_PSCR,  KC_SLCK,  KC_PAUS,
+    KC_ESC,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,  KC_INS,   KC_HOME,  KC_PGUP,
+    KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,  KC_BSLS,  KC_DEL,   KC_END,   KC_PGDN,
+    KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,            KC_ENT,
+    KC_LSFT,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,            KC_RSFT,                      KC_UP,
+    KC_LCTL,            KC_LALT,                      KC_SPC,                                                     KC_RALT,            KC_RCTL,  KC_LEFT,  KC_DOWN,  KC_RGHT
+  ),
+
+  [1] = LAYOUT_all(
```
Comment (fauxpark) [inline keyboards/hardlineworks/otd_plus/keymaps/default/keymap.c:14]:
```suggestion
  [1] = LAYOUT_tkl_ansi_wkl(
```

File: keyboards/hardlineworks/otd_plus/info.json [hunk 6]
```diff
@@ -0,0 +1,103 @@
+{
+    "keyboard_name": "OTD-PLUS",
+    "url": "",
+    "maintainer": "Hardlineworks",
+    "width": 18.25,
+    "height": 6.25,
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (fauxpark) [inline keyboards/hardlineworks/otd_plus/info.json:8]:
```suggestion
        "LAYOUT_tkl_ansi_wkl": {
```

File: keyboards/hardlineworks/otd_plus/info.json [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+                {"label" : "F5", "x":6.5, "y":0},
+                {"label" : "F6", "x":7.5, "y":0},
+                {"label" : "F7", "x":8.5, "y":0},
+                {"label" : "F8", "x":9.5, "y":0},
+                {"label" : "F9", "x":11, "y":0},
+                {"label" : "F10", "x":12, "y":0},
+                {"label" : "F11", "x":13, "y":0},
+                {"label" : "F12", "x":14, "y":0},
+                {"label" : "Print screen", "x":15.25, "y":0},
+                {"label" : "Scrool lock", "x":16.25, "y":0},
```
Comment (fauxpark) [inline keyboards/hardlineworks/otd_plus/info.json:25]:
```suggestion
                {"label" : "Scroll lock", "x":16.25, "y":0},
```

**Comment by gadzkun** [comment]
done @fauxpark @zvecr

File: keyboards/hardlineworks/otd_plus/config.h [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x484C  // "HL"
+#define PRODUCT_ID      0x0087  // "TKL"
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Hardlineworks
+#define PRODUCT         otd plus
+#define DESCRIPTION     TKL Mechanical Keyboard
```
Comment (fauxpark) [inline keyboards/hardlineworks/otd_plus/config.h:11]:
```suggestion
```

File: keyboards/hardlineworks/otd_plus/config.h [hunk 9]
```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x484C  // "HL"
+#define PRODUCT_ID      0x0087  // "TKL"
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Hardlineworks
+#define PRODUCT         otd plus
```
Comment (fauxpark) [inline keyboards/hardlineworks/otd_plus/config.h:10]:
```suggestion
#define PRODUCT         OTD-PLUS
```

Comment (gadzkun) [reply keyboards/hardlineworks/otd_plus/config.h]: done @fauxpark

**Comment by gadzkun** [comment]
@zvecr any some update for this PR?

**Pull request merged**